requirements/base.txt

```diff
@@ -9,7 +9,7 @@ confuse==2.0.0
     # via -r requirements/base.in
 graphlib-backport==1.0.3
     # via -r requirements/base.in
-numpy==1.23.1
+numpy==1.23.2
```

Comment:
We should avoid big requirement updates as part of unrelated PRs. I suggest to first do an update independently, then introduce the new dependencies.

Reply:
How would we do this? We need to add a dependency. Can we lock in such a way that everything else stays the same?

Reply:
So should I open a separate PR for the version updates or leave it as it is.

Reply:
Open a separate PR.